en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -306,17 +310,17 @@ And the following configuration..
 
 ```yml
 ---
-name: MyController
+name: App\MyController
```

Comment:
not sure how backslashes work in yaml section names.. would remove it here or put the name in quotes